### PR TITLE
Virtual Box 4.2 support

### PR DESCRIPTION
**Pull request by cwerner1**
I upgraded the vagrant version 1.0.6.dev , which works VirtualBox 4.2.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,11 +3,14 @@ source :gemcutter
 gem "jenkins-plugin-runtime"
 gem "jenkins-plugin", '~> 0.2.0'
 #gem "vagrant", '~> 1.0.1'
-gem 'vagrant', :git => 'git://github.com/rtyler/vagrant', :ref => '8a5b1ad'
+#gem 'vagrant', :git => 'git://github.com/rtyler/vagrant', :ref => '8a5b1ad'
+#gem 'vagrant', :git => 'git://github.com/cwerner/vagrant', :ref => '8a5b1ad'
+gem 'vagrant', :git => '/home/christian/src/vagrant', :ref => '95d313c'
```
Comment (rtyler) [inline Gemfile:8]: I don't think I can merge this in since `/home/christian/src/vagrant` doesn't exist on my machine ;)

File: Gemfile [hunk 2]
```diff
@@ -3,11 +3,14 @@ source :gemcutter
...
 gem 'jruby-openssl'
 
 group :development do
-  gem 'jpi', '~> 0.3.3'
+  #, '~> 0.3.3'
+  gem 'jpi'
```
Comment (rtyler) [inline Gemfile:13]: Why remove the version pinning for the jpi gem? Is there a more recent release that has new goodies?

**Comment by vStone** [comment]
~~afaik vagrant stable also supports virtualbox 4.2. I would close this ;)~~

I was thinking about the wrong project.. ignore pls :/

**Comment by brejoc** [comment]
Should this be available via jenkins update center? How would I enable this?

**Comment by rtyler** [comment]
@brejoc This hasn't been merged yet, so no change will be available yet. Unfortunately I've been pretty busy lately and haven't been able to fix up and test this change myself :(

**Comment by duro** [comment]
@rtyler Going to get around to getting this fixed? I think this plugin is going to find itself used more and more as Vagrant continues to work it's way in to many developers workflows. I would either find some time to keep maintaining, or get some co-maintainers :)

**Comment by rtyler** [comment]
This is no longer applicable, I've got an updated version of the plugin in `master` but I can't yet release it due to plugin compat issues I can't yet figure out.

**Pull request closed without merging**